src/ui/control/geolocate_control.js

```diff
@@ -78,6 +80,7 @@ let noTimeout = false;
  * @param {Object} [options.positionOptions={enableHighAccuracy: false, timeout: 6000}] A Geolocation API [PositionOptions](https://developer.mozilla.org/en-US/docs/Web/API/PositionOptions) object.
  * @param {Object} [options.fitBoundsOptions={maxZoom: 15}] A [`fitBounds`](#map#fitbounds) options object to use when the map is panned and zoomed to the user's location. The default is to use a `maxZoom` of 15 to limit how far the map will zoom in for very accurate locations.
  * @param {Object} [options.trackUserLocation=false] If `true` the Geolocate Control becomes a toggle button and when active the map will receive updates to the user's location as it changes.
+ * @param {Object} [options.showAccuracyRadius=true] Show a transparent circle approximating the position accuracy as reported by `navigator.geolocation.watchPosition`. Set to `false` to disable.
```

Comment:
How about "By default a transparent circle will be drawn around the user location indicating the accuracy of the user's location"? What do you think?

I don't think we need to mention watchPosition,

Reply:
I see this as just an extension of the showUserLocation option, should we support showUserLocation=false but still showing this accuracy circle? I'm not sure...

I'm on the fence with the default. After all we introduced showUserLocation=true as the default which changed behaviour when people upgraded. While I can understand we should try to avoid to unexpected changes in behaviour for users upgrading, this change is mostly cosmetic and if you're already showing the user location dot would you ever not want to show this accuracy circle?

Reply:
Yeah I think this is the core question -- are there use cases for disabling the circle entirely? I do think defaulting to it visible makes sense although it is a change. I added the flag anticipating someone might see the change and want to go back to the old behavior (especially if they have done something unusual like customized the dot icon or etc).

I do think there is some value in explaining "how is the size of this circle chosen" in the docs, but can leave it out if there's agreement it isn't needed.

We could name it something friendly: "circle-of-confusion" is common but a bit wordy... the radius shown specifically represents the 95% confidence level in the latitude/longitude.

Maybe in this context just `showAccuracy` is sufficient?

Reply:
Updated to
> By default a transparent circle will be drawn around the user location indicating the accuracy (95% confidence level) of the user's location. Set to `false` to disable.

for now.